### DISCUSSION TITLE
Enable [control] + [left mouse] click

### DISCUSSION THREAD

**Discussion by Richard-97** [nick-keller/react-datasheet-grid#165] [Feature requests]
Hello,

I would like to ask you, if it's possible to somehow enable [control] + [left click] to behave just like mouse right click. 
I checked the `onKeyDown` function in  `DataSheetGrid.tsx` file, but I didn't find some prevention on it.

Thanks for the answer,
Richard

**Comment by nick-keller** [comment]
That sounds like a reasonable feature, is this to mimic the way apple handle right click?

**Comment by nick-keller** [comment]
@Richard-97 This is now available in 4.6.0
